backend/src/Designer/Controllers/OptionsController.cs

```diff
@@ -251,4 +255,27 @@ public async Task<ActionResult> DeleteOptionsList(string org, string repo, [From
 
         return Ok($"The options file {optionsListId}.json has been deleted.");
     }
+
+    [HttpPost]
+    [Route("{optionsListId}/import")]
+    public async Task<ActionResult> ImportOptionsListFromOrg(string org, string repo, [FromRoute] string optionsListId, CancellationToken cancellationToken = default)
```

Comment:
Jeg tror vi burde bruke `OptionList` overalt i løsningen vår.
```suggestion
    public async Task<ActionResult> ImportOptionListFromOrg(string org, string repo, [FromRoute] string optionListId, CancellationToken cancellationToken = default)
```